alerts/openshift-container-storage-operator/helpers/sre-to-engineering-escalation.md

```diff
@@ -0,0 +1,141 @@
+
+## Purpose
+
+This SOP describes how the SRE Team will escalate issues to the
+ODF Engineering Team in case of incidents which would require help
+from ODF engineering to resolve them.
+
+## Scope
+
+The scope of this escalation path is to cover the incidents which have
+unavailability/data loss associated with the customer and none of the
+existing SOPs help to resolve the respective incident at the same time.
+
+## Prerequisites
+
+* `MTSRE` Role permissions over OCM.
+
+## Responsibilities
+
+* SRE to only escalate the issue to engineering on an urgent basis when the
+incident is associated with unavailability or data loss for the customer.
+* ocm-cli version 0.1.63
+
+## Procedure
+
+### Determine whether the incident leads to unavailability / data loss for the customer
+
+The best way to determine this is to see if the corresponding provider cluster
+has any storageconsumers or not.
+
+* Determine the cluster ID of the Provider cluster associated with the customer,
+in case it's not directly available from PagerDuty incident: Happens when the
+incident's source is an ocs-consumer.
+
+* Backplane access into the provider cluster (Checkout [References](#references)
+for the SOP to do so).
+
+* See if there are any storageconsumers or not.
+
+    oc get storageconsumers.ocs.openshift.io -n openshift-storage
+
+If there are no storageconsumers, then wait until the next set of ODF's working
+hours to escalate the issue to engineering.
+
+### Escalation path when the incident urgently requires help from ODF Engineering
+
+* Follow the steps in [link](<https://source.redhat.com/groups/public/openshiftplatformsre/wiki/how_to_sre_escalate_towards_engineering>),
+starting with submitting the Google Form,
+to get help from the ODF Engineering Team.
+
+### If above steps do not work
+
+Follow the manual steps to escalate to ODF Engineering Team
+
+* Fetch certain details of the impacted cluster:
+  * Cluster Details:
+
+    ocm describe cluster <internal/external-id-of-cluster>
+
+    For example
+
+    ```bash
+    ❯ ocm describe cluster 1s0q9mp65e2f370q9s7ip75g6sj0sifg
+
+    ID:      1s0q9mp65e2f370q9s7ip75g6sj0sifg
+    External ID:    7ca71103-f0d8-4e66-a586-35332eaa4b10
+    Name:      e2e-ci-prov
+    State:      ready
+    API URL:    https://api.e2e-ci-prov.mngt.s1.devshift.org:6443
+    API Listening:    external
+    Console URL:    https://console-openshift-console.apps.e2e-ci-prov.mngt.s1.devshift.org
+    Masters:    3
+    Infra:      2
+    Computes:    3
+    Product:    osd
+    Provider:    aws
+    Version:    4.10.11
+    Region:      us-west-2
+    Multi-az:    false
+    CCS:      true
+    Subnet IDs:    [subnet-06a34a7bb480707bf subnet-07b3053eb531c5e70]
+    PrivateLink:    false
+    STS:      false
+    Existing VPC:    true
+    Channel Group:    stable
+    Cluster Admin:    true
+    Organization:    Red Hat-Storage-OCS
+    Creator:    almartin-storage-ocs
+    Email:      almartin@redhat.com
```

Comment:
Please: Remove "real" and "personal"  data from documentation

Reply:
Done